openedx_learning/core/publishing/api.py

```diff
@@ -131,7 +135,7 @@ def publish_from_drafts(
             learning_package_id=learning_package_id,
             message=message,
             published_at=published_at,
-            published_by=published_by,
+            published_by_id=published_by,
```

Comment:
This was a bug: the function accepted an `int` but assigned it to a `User` model attribute.